packages/connect-explorer/src/js/components/Devices.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useActions, useSelector } from '../hooks';
+import * as trezorConnectActions from '../actions/trezorConnectActions';
+
+const Nav = styled.nav`
+    position: fixed;
+    top: 50px;
+    width: 100%;
+    z-index: 100;
+`;
+
+const LayoutWrapper = styled.div`
+    color: #fff;
+    background: #2c2c2c;
+    padding: 0;
+`;
+
+const DeviceList = styled.ul`
+    list-style: none;
+`;
+
+const DeviceItem = styled.li`
+    position: relative;
+    display: block;
+    cursor: pointer;
+    padding: 10px 15px;
+    white-space: nowrap;
+    //overflow: hidden;
+    width: 25%;
+    display: inline-block;
+    border-top: 1px solid transparent;
+    border-bottom: 4px solid transparent;
+    &.active {
+        background: #060606;
+        border-top-color: #2c2c2c;
+        border-bottom-color: #4cc148;
+    }
+
+    &.unacquired {
+        color: gray;
+    }
+
+    &.used-elsewhere {
+        color: red !important;
+    }
+
+    &.reload-features {
+        color: orange;
+    }
+`;
+// ul {
```

Comment:
why is this commented? Please add  comment or remove it

Reply:
good catch. code is transfered from another repo. many things like this might already been there and I have might missed them.